code/modules/mob/living/silicon/silicon.dm

```diff
@@ -24,7 +24,7 @@
 	var/obj/item/camera/siliconcam/aicamera = null //photography
 	hud_possible = list(ANTAG_HUD, DIAG_STAT_HUD, DIAG_HUD, DIAG_TRACK_HUD)
 
-	var/obj/item/radio/borg/radio = null //All silicons make use of this, with (p)AI's creating headsets
+	var/obj/item/radio/borg/pai/radio = null //All silicons make use of this, with (p)AI's creating headsets
```

Comment:
This is for all silicons
you can assign a pAI radio to it but don't change the base type

Reply:
fixed